src/main/java/com/openlattice/conductor/rpc/ConductorElasticsearchApi.java

```diff
@@ -143,7 +143,8 @@ boolean createEntityData(
 
     boolean createBulkLinkedData(
             UUID entityTypeId,
-            Map<UUID, Map<UUID, Map<UUID, Set<Object>>>> entitiesByLinkingId );
+            Map<UUID, Map<UUID, Map<UUID, Map<UUID, Set<Object>>>>> entitiesByLinkingId
```

Comment:
can you document somewhere what all these UUIDs represent?

Reply:
sure!

Reply:
JavaDoc or KDoc please